music21/musicxml/xmlToM21.py

```diff
@@ -3645,6 +3644,19 @@ def flatten(mx, name):
                 fermata.shape = mxObj.text.strip()
             n.expressions.append(fermata)
 
+        # get any arpeggios, store in expressions.
+        for mxObj in mxNotations.findall('arpeggiate'):
+            arpeggioType: str = mxObj.get('direction')
+            if not arpeggioType:
+                arpeggioType = 'normal'
```

Comment:
The constructor defaults to 'normal', so I'm not certain you need this check here. For consistency you could just use `textStripValid` like the surrounding cases. (I'm guessing that some musicxml writers at some point in time were adding extra whitespace, but I don't have direct experience with that.)

Reply:
I've just removed the check.  I don't use mxObj.text here, so I'm not sure how textStripValid will apply.

Reply:
And I've changed the constructors to take None (and map it to 'normal').